lucida/commandcenter/controllers/Config.py

```diff
@@ -12,41 +12,163 @@
 # models are already saved in `../models`.
 TRAIN_OR_LOAD = 'train' # either 'train' or 'load'
 
+
```

Comment:
Hm...I think this is something I need to think about in the general proposal I have for Lucida.

Lucida should never even have trained models (Besides the classifier/maybe ASR?). This would be exclusive to the blackboxes.